src/puppetlabs/trapperkeeper/services/metrics/metrics_core.clj

```diff
@@ -90,6 +93,20 @@
                 (fn [req]
                   (ringutils/json-response 200
                                            (metrics-utils/mbean-names))))
+
+              (comidi/POST "" []
+                (fn [req]
+                  (try
+                    (let [metrics (with-open [reader (-> req :body io/reader)]
+                                    (json/parse-stream reader true))]
+                      (if (seq? metrics)
+                        (ringutils/json-response
+                         200 (map metrics-utils/get-mbean metrics))
```

Comment:
I think it might be better to return a map here wherein each key is the metric name and each value is the value of that metric rather than returning a straight array.